cmd/root.go

```diff
@@ -146,6 +147,9 @@ func setLogLevel() {
 	if debug {
 		zerolog.SetGlobalLevel(zerolog.DebugLevel)
 	}
+	if runtime.GOOS == "windows" {
+		log.Output(zerolog.ConsoleWriter{Out: color.Output})
+	}
```

Comment:
Do you think that this should move to main.go ?

Reply:
I'm a little confused by this should only apply to windows. If I'm reading correctly, [`color.Output`](https://github.com/fatih/color/blob/master/color.go#L25) = `colorable.NewColorableStdout()`, which should be Stdout regardless of OS right? Same comment for https://github.com/get-woke/woke/pull/20/files#diff-7f2a90cc6cb2d275be21c4c36d18055821064a64d15b3fccd53e1641c77a03cdR33-R35

Reply:
I'll use mattn/go-colorable directly. Thanks.

Reply:
Works for me, thanks! Appreciate the contribution... it made me realize I should be running CI tests against windows as well so I'll be looking into that.